host/CMakeLists.txt

```diff
@@ -488,3 +488,321 @@ install(TARGETS oehostapp EXPORT openenclave-targets)
 install(TARGETS oehostverify EXPORT openenclave-targets
   ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}/openenclave/host
   COMPONENT OEHOSTVERIFY)
+
+# Add a library oemr independent of libsgx_enclave_common.so/libsgx_dcap_ql.so
+if (UNIX)
+	set(PLATFORM_OEMR_SRC
+		#    ../common/cert.c
+		#    crypto/openssl/asn1.c
```

Comment:
Why are these commented out?

Reply:
I agree with @radhikaj. These commented lines could be removed.

Reply:
Thanks, I will clean up when everything is done.